### ISSUE TITLE
CIFAR-10 task performance in the paper can't be reproduced

### ISSUE THREAD

**Issue by lndezxn** [YWolfeee/Training-Free-Guidance#9]
Thank you for your remarkable work!
When I run the script _cifar10_label.sh_, the result seems much worse than the data mentioned in the paper, and image generated seems corrupted, with over-saturated noise everywhere.
I strictly followed the hyperparameter selection in Table 11 for each target, but still can't get satisfying output.

_metrics.json_ of each target:
{"inception_score": 4.786691188812256, "fid": 137.68205178369456, "validity": 0.890625, "validity_resnet_cifar10.pt": 0.890625}
{"inception_score": 4.5060625076293945, "fid": 170.29827268373225, "validity": 0.3828125, "validity_resnet_cifar10.pt": 0.3828125}
{"inception_score": 5.237436294555664, "fid": 201.56453484922935, "validity": 0.4609375, "validity_resnet_cifar10.pt": 0.4609375}
{"inception_score": 6.1551337242126465, "fid": 185.94133839845017, "validity": 0.453125, "validity_resnet_cifar10.pt": 0.453125}
{"inception_score": 6.019697189331055, "fid": 203.12209198417568, "validity": 0.390625, "validity_resnet_cifar10.pt": 0.390625}
{"inception_score": 5.850213527679443, "fid": 188.02872927604534, "validity": 0.484375, "validity_resnet_cifar10.pt": 0.484375}
{"inception_score": 4.634642601013184, "fid": 242.29437638940945, "validity": 0.6796875, "validity_resnet_cifar10.pt": 0.6796875}
{"inception_score": 5.1896586418151855, "fid": 156.66782199956748, "validity": 0.46875, "validity_resnet_cifar10.pt": 0.46875}
{"inception_score": 3.1600444316864014, "fid": 100.10934208159887, "validity": 0.84375, "validity_resnet_cifar10.pt": 0.84375}
{"inception_score": 3.4470088481903076, "fid": 121.25581320330042, "validity": 0.78125, "validity_resnet_cifar10.pt": 0.78125}

**Comment by linhaowei1** [comment]
Hello,

I reviewed the current script used for CIFAR-10 label guidance (target = 8), and my results are similar to yours (FID ~100, validity ~0.84). It seems unlikely that you ran the code incorrectly. The image quality for CIFAR-10 label guidance is generally not very high, and the baselines perform even worse. This reinforces our goal of establishing a unified benchmark to highlight the challenge of training-free guidance.

If you’re aiming to reproduce the paper’s results, I recommend increasing the number of samples to 2,560 instead of the 128 used in the current script, as FID is highly sensitive to sample size.

**Comment by lndezxn** [comment]
Increasing the sample size has made FID more stable. Thanks a lot!
